core/conn/jdbcT4/src/main/java/org/trafodion/jdbc/t4/ExtractLobReply.java

```diff
@@ -0,0 +1,56 @@
+// @@@ START COPYRIGHT @@@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+// @@@ END COPYRIGHT @@@
+
+package org.trafodion.jdbc.t4;
+
+import java.nio.charset.CharacterCodingException;
+import java.nio.charset.UnsupportedCharsetException;
+import java.sql.SQLException;
+
+class ExtractLobReply {
+	int totalErrorLength;	
+	SQLWarningOrError[] errorList;
+
+	odbc_SQLSvc_ExtractLob_exc_ m_p1;
+	String m_p2;
+	SQLItemDescList_def m_p3;
+	ERROR_DESC_LIST_def m_p4;
+	String proxySyntax = "";
```

Comment:
I don't see members m_p2, m_p3, m_p4 and proxySyntax used anywhere. Do we need them?